### PR TITLE
DOCS-59 Add release notes guidelines to the style guide

### PR DESCRIPTION
**Pull request by patrickbirch**
A  docs/structure-release-notes.md

### CODE REVIEW

**Review by nastena1606 — APPROVED** [review]

File: docs/structure-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Release notes structure guidelines
+
+The release note filename should follow the file naming guidelines.
```
Comment (alina-derkach-oaza) [inline docs/structure-release-notes.md]: release note -> notes

Comment (patrickbirch) [reply docs/structure-release-notes.md]: Fixed

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

File: docs/structure-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Release notes structure guidelines
+
+The release notes filename should follow the file naming guidelines.
```
Comment (LenzGr) [inline docs/structure-release-notes.md]: Would it be possible to link to these guidelines here?

Comment (patrickbirch) [reply docs/structure-release-notes.md]: Added a doc for file naming and linked it.

**Review by LenzGr — APPROVED** [review]
LGTM! Thanks a lot for adding this.

**Review by alexfiru — APPROVED** [review]

**Pull request merged**